engine/orchestration/src/com/cloud/agent/manager/AgentManagerImpl.java

```diff
@@ -821,9 +819,9 @@ protected boolean handleDisconnectWithInvestigation(final AgentAttache attache,
             try {
                 nextStatus = host.getStatus().getNextStatus(event);
             } catch (final NoTransitionException ne) {
-                /* Agent may be currently in status of Down, Alert, Removed, namely there is no next status for some events.
-                 * Why this can happen? Ask God not me. I hate there was no piece of comment for code handling race condition.
-                 * God knew what race condition the code dealt with!
+                /*
```

Comment:
@miguelaferreira Could you please remove this comment (or, at least, this part : "Why this can happen? Ask God not me..." )? I don't think that this comment is necessary. Thanks